<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cImportDialog</class>
 <widget class="QDialog" name="cImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1000,1,0,0,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lpPath"/>
       </item>
       <item>
        <widget class="QPushButton" name="m_lpPathSelect">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="m_lpRecursive">
           <property name="text">
            <string>recursive</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QCheckBox" name="m_lpPictures">
           <property name="text">
            <string>pictures</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_lpVideos">
           <property name="text">
            <string>videos</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="m_lpCopy">
           <property name="text">
            <string>copy</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">copy_or_move</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="m_lpMove">
           <property name="text">
            <string>move</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">copy_or_move</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="m_lpDetectHDR">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>detect HDR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_lpDetectJPG_RAW">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>detect JPG/RAW</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="m_lpRead">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>read</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_lpImport">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>import</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="m_lpSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="m_lpFolderView">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QListView" name="m_lpThumbnailView">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>160</width>
          <height>120</height>
         </size>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="gridSize">
         <size>
          <width>180</width>
          <height>180</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>69</width>
           <height>315</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="cToolBoxInfo" name="m_lpInfo" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="m_lpCount">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
       <item>
        <widget class="QLabel" name="m_lpStatusText">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="m_lpProgress">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="m_lpTotalProgress">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="m_lpButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cToolBoxInfo</class>
   <extends>QWidget</extends>
   <header>ctoolboxinfo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_lpPath</tabstop>
  <tabstop>m_lpPathSelect</tabstop>
  <tabstop>m_lpRecursive</tabstop>
  <tabstop>m_lpPictures</tabstop>
  <tabstop>m_lpVideos</tabstop>
  <tabstop>m_lpCopy</tabstop>
  <tabstop>m_lpMove</tabstop>
  <tabstop>m_lpDetectHDR</tabstop>
  <tabstop>m_lpDetectJPG_RAW</tabstop>
  <tabstop>m_lpRead</tabstop>
  <tabstop>m_lpImport</tabstop>
  <tabstop>m_lpFolderView</tabstop>
  <tabstop>m_lpThumbnailView</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>cImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>cImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="copy_or_move"/>
 </buttongroups>
</ui>
